desktop/src/main/java/bisq/desktop/util/GUIUtil.java

```diff
@@ -1115,6 +1115,8 @@ public static String getBsqInUsd(Price bsqPrice,
     }
 
     public static MaterialDesignIcon getIconForSignState(AccountAgeWitnessService.SignState state) {
+        if (state.equals(AccountAgeWitnessService.SignState.PEER_INITIAL))
+            return MaterialDesignIcon.CLOCK;
```

Comment:
We (and the compiler) know, but there should(?) be a white space after the one line inner block of the if statement.

Reply:
thanks for the reminder, I've been warned before about my `if` statements!  :smiley: